permits/templates/permits/_permit_request_summary.html

```diff
@@ -109,5 +109,44 @@ <h4>{% trans "Éléments géographiques touchés par la demande (automatique)" %
     </div>
   </div>
 
+  {% if documents %}
+    <div class="row">
+      <div class="col">
+        <h2>Documents</h2>
+
+        {% for document in documents %}
+          <dl class="mb-4">
+            <dt>{% trans "Nom" %}</dt>
+            <dd>
+                <div class="pl-0 col-6">
+                  {{ document.document }}
+                  {% if document.is_public %}
+                    <span class="badge badge-pill badge-success">{% trans "Publique" %}</span>
```

Comment:
The pill-badge makes the text look very small in the summary page, maybe make it bigger?